module/mouse_gutbrain_de_independent_guide.qmd

```diff
@@ -261,34 +268,134 @@ ggplot(data = asd_vs_c, aes(x = log2FoldChange, y = padj_trans)) +
          geom_point()
 ```
 
-We can also colorcode the genes (points) which are significantly upregulated or downregulated. In order to do this, we will need to create a new column in our dataset that identifies whether a gene is upregulated in ASD-type mice, downregulated in ASD-type mice, or has the same regulation in ASD-type mice compared to control-type mice.
+We can also color code the genes (points) which are significantly upregulated or downregulated. In order to do this, we will need to create a new column in our dataset that identifies whether a gene is upregulated in ASD-type mice, downregulated in ASD-type mice, or has the same regulation in ASD-type mice compared to control-type mice.
+
+In this new column, we can decide the thresholds that a `log2FoldChange` value must meet in order to be considered upregulated or downregulated. We will consider genes to be upregulated if the log2fold change is greater than 0.6 and downregulated if it is less than -0.6. **You can change these thresholds to whatever makes sense for you.**
```

Comment:
technically positive or negative? But we want to make sure it's "real" so we want to look on the high and low end. Might want to explain that the log2foldchange is just a representation of a ratio